semesterly/celery_conf.py

```diff
@@ -15,11 +15,11 @@
 from django.conf import settings
 
 # set the default Django settings module for the 'celery' program.
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'semesterly.settings')
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "semesterly.settings")
```

Comment:
Is this is autoformatter? 
This is fine but it would be nice to have a consistent style of double or single quotes across all py files.

Reply:
Yes, this is the first commit and part of the style guide I'm making